src/test/java/org/mule/test/http/functional/listener/HttpListenerConfigFunctionalTestCase.java

```diff
@@ -99,7 +99,7 @@ public void listenerConfigOverridesListenerConfig() throws Exception {
   public void noListenerConfig() throws Exception {
     final String url = format("http://localhost:%s/noListener", noListenerConfigPort.getNumber());
     final HttpResponse httpResponse = callAndAssertStatus(url, NOT_FOUND);
-    assertThat(IOUtils.toString(httpResponse.getEntity().getContent()), is(format(NO_LISTENER_ENTITY_FORMAT, "/noListener")));
+    assertThat(IOUtils.toString(httpResponse.getEntity().getContent()), is(format(NO_LISTENER_ENTITY_FORMAT, url)));
```

Comment:
Same as in https://github.com/mulesoft/mule-http-service/pull/56#discussion-diff-148074617R77